### ISSUE TITLE
Actions: If an assignee is assigned to an issue, apply "in progress" label to it

### ISSUE THREAD

**Issue by Maelstromeous** [satisfactory-factories/application#139]
This usually indicates that they are intending to work on the issue, while it may not fully mean they're working on it right that instance, they will likely do so soon.

**Comment by samsmithnz** [comment]
isn't this exactly the same as having someone assigned? Does the assignee make the label redundant? 

I would like to automatically move the issue to the 'in progress' column in the project board - but this is currently still a feature request.

**Comment by Maelstromeous** [comment]
Yeah that would be probably more sensible than a label to be fair
